src/main/webapp/app/scores/exercise-scores.component.ts

```diff
@@ -36,6 +36,7 @@ export class ExerciseScoresComponent implements OnInit, OnDestroy {
     results: Result[];
     allResults: Result[];
     eventSubscriber: Subscription;
+    newResultAllowed: boolean;
```

Comment:
please rename to newManualResultsAllowed

Reply:
Renamed it